packages/edit-post/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -36,4 +36,14 @@ export const textFormattingShortcuts = [
 		keyCombination: { modifier: 'access', character: 'x' },
 		description: __( 'Make the selected text inline code.' ),
 	},
+	{
+		keyCombination: { modifier: 'access', character: 'Level' },
```

Comment:
I wonder if instead of `'Level'` it could be `'1-6'`

Reply:
For some reason it can't

Reply:
Oh the thing again, in the function that displays shortcut, I hit that strip regexp back when added `[[` :)